src/pages/markets/static/content/_margin.js

```diff
@@ -12,14 +12,13 @@ import {
     StepIndices,
     VolatilityIndices,
 } from '../../instruments/_submarkets'
-import { smart_gold_index, step_fx } from '../../instruments/_market-symbols'
+import { smart_gold_index } from '../../instruments/_market-symbols'
 import {
     CrashBoomDetails,
     EuropeDetails,
     RangeBreakIndicesDetails,
     SmartFXDetails,
     SmartGoldIndexDetails,
-    StepFXDetails,
```

Comment:
I think you may have forgotten to remove `StepFXDetails` from `markets/static/content/_details.js` as well.